pkg/system/reconciler.go

```diff
@@ -223,6 +227,9 @@ func NewReconciler(
 	r.SecretServer.StringData["jwt"] = util.RandomBase64(16)
 	r.SecretServer.StringData["server_secret"] = util.RandomHex(4)
 
+	r.SecretDB.StringData["user"] = "noobaa"
+	r.SecretDB.StringData["password"] = util.RandomBase64(10)
```

Comment:
if you generate the password here won't it change the password in the secret when NewReconciler is called?

Reply:
@dannyzaken I don't think it suppose to change anything - like in jwt and server_secret - correct me if Im wrong here...

Reply:
I mean it will change in memory but not the secret itself

Reply:
you're right. the `kubeCheck` will load the existing secret if present